package.json

```diff
@@ -9,6 +9,8 @@
     "dev": "webpack-dev-server --https --inline --progress --config build/webpack-dev-conf.js",
     "start": "npm run dev",
     "unit": "jest --config test/unit/jest-conf.js --coverage",
+    "qa-install": "brew install geckodriver && brew install pipenv",
```

Comment:
Shouldn't this include `&& cd test/selenium-webdriver && pipenv install`?

Reply:
That step is executed with `pipenv run`.